File: SEM3_CODE/PHP/BASIC/21_star_pattern.php
<?php
$size = (int) readline('Enter a size: ');          // Read the number of lines

for ($line = 1; $line <= $size; $line++) {         // Create each pattern line
    $stars = '';                                   // Start with an empty line
    for ($count = 1; $count <= $line; $count++) {  // Add stars one by one
        $stars = $stars . '*';                     // Add one star
    }
    echo $stars . PHP_EOL;                         // Display the pattern line
}
?>

File: SEM5_CODE/PYTHON/BASIC/21_star_pattern.py
size = int(input("Enter a size: "))  # Read the number of lines

for line in range(1, size + 1):      # Create each pattern line
    stars = ""                       # Start with an empty line
    for count in range(line):        # Add stars one by one
        stars = stars + "*"          # Add one star
    print(stars)                     # Display the pattern line
